lib/nylas/resources/grants.rb

```diff
@@ -13,9 +13,9 @@ class Grants < Resource
     # Return all grants.
     #
     # @param query_params [Hash, nil] Query params to pass to the request.
-    # @return [Array(Array(Hash), String)] The list of grants and API Request ID.
+    # @return [Array(Array(Hash), String, String)] The list of grants, API Request ID, and next cursor.
     def list(query_params: nil)
-      get(
+      get_list(
```

Comment:
I don't think grants endpoints uses cursor pagination, this would be limit/offset pagination.